src/vaadin-context-menu.html

```diff
@@ -333,23 +336,34 @@
              * - `context` The object with the menu context, contains:
              *   - `context.target`  the target of the menu opening event,
              *   - `context.detail` the menu opening event detail.
+             * @type {ContextMenuRenderer | undefined}
              */
             renderer: {
               type: Function
             },
 
+            /** @private */
             _context: Object,
 
+            /** @private */
             _boundClose: Object,
 
+            /** @private */
             _boundOpen: Object,
 
+            /**
+             * @type {HTMLTemplateElement}
+             * @protected
+             */
             _contentTemplate: Object,
```

Comment:
Why not private? Does it go hand in hand with `_setTemplateFromNodes`?

Reply:
Yes, this one is related to that method. I can make them both private for now.

Reply:
I'm ok with either approach. @pekam can resolve.